### PR TITLE
private docker registry doc

### PR DESCRIPTION
**Pull request by sascala**
## Description
https://jira.mesosphere.com/browse/DOCS-1733

Note: once this page is live, the tooltip in the UI needs to be updated: https://jira.mesosphere.com/browse/DCOS_OSS-906

## Urgency
- [ ] Blocker 
- [X] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by sascala** [comment]
@Gilbert88 I'm pretty sure this doc is correct -- I was able to test it successfully until the last step, when my cluster gave me this error: `
Failed to launch container: Failed to fetch all URIs for container '89fcb0a5-fa19-4715-954e-ad893e507253' with exit status: 256`

I'm assuming this is an error or oversight on my part?

File: 1.9/usage/managing-services/private-docker-registry.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      Email: foo@bar.com
+    ```
+
+1. Tar this folder and its contents.
+
+    ```bash
+    cd ~
+    tar -czf docker.tar.gz .docker
+    ```
+1. Check you have both files in the tar.
```
Comment (joel-hamill) [inline 1.9/usage/managing-services/private-docker-registry.md]: `Verify that both files are in the archive file. `

File: 1.9/usage/managing-services/private-docker-registry.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+post_title: Using a Private Docker Registry 
+menu_order: 004.5
+---
+
```
Comment (joel-hamill) [inline 1.9/usage/managing-services/private-docker-registry.md]: This topic needs an introduction. Right now it immediately jumps into the task without context. 
- where does the `docker.tar.gz` file come from?
- where does the `uris` field come from? what app? is this a marathon app definition parameter?

File: 1.9/usage/managing-services/private-docker-registry.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+1. Log in to the private registry manually. Login creates a `.docker` folder and a `.docker/config.json` file in your home directory.
+
+    ```bash
+      docker login some.docker.host.com
+      Username: foo
+      Password:
+      Email: foo@bar.com
+    ```
+
+1. Tar this folder and its contents.
```
Comment (joel-hamill) [inline 1.9/usage/managing-services/private-docker-registry.md]: Compress the `.docker` folder and its contents.

File: 1.9/usage/managing-services/private-docker-registry.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+1. Check you have both files in the tar.
+
+    ```bash
+      tar -tvf ~/docker.tar.gz
+
+      drwx------ root/root         0 2015-07-28 02:54 .docker/
+      -rw------- root/root       114 2015-07-28 01:31 .docker/config.json
+    ```
+
+1. Put the gzipped file in location that you will then reference in your application definition.
```
Comment (joel-hamill) [inline 1.9/usage/managing-services/private-docker-registry.md]: `Put the archive file in a location that is accessible by your application definition.`

File: 1.9/usage/managing-services/private-docker-registry.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+    ```bash
+    $ cp docker.tar.gz /etc/
+    ```
+
+    **Important:** The URI you create must be accessible by all nodes that may start your application. You can distribute the
```
Comment (joel-hamill) [inline 1.9/usage/managing-services/private-docker-registry.md]: `The URI must be accessible by all nodes that will start your application. `

File: 1.9/usage/managing-services/private-docker-registry.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+file to the local filesystem of all nodes, for example via RSYNC/SCP, or store it on a shared network drive like [Amazon
+S3](http://aws.amazon.com/s3/). Consider the security implications of your chosen approach carefully.
+
+### Step 2: Mesos/Marathon config
+
+1. Add the path to the gzipped login credentials to your Marathon app definition
```
Comment (joel-hamill) [inline 1.9/usage/managing-services/private-docker-registry.md]: `Add the path to the archive file login credentials to your app definition.`

File: 1.9/usage/managing-services/private-docker-registry.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+          "network": "HOST"
+        }
+      },
+      "uris":  [
+          "file:///etc/docker.tar.gz"
+      ]
+    }
+    ```
+
+1. The Docker image will now pull using the security credentials you provided.
```
Comment (joel-hamill) [inline 1.9/usage/managing-services/private-docker-registry.md]: `... that you specified.`

**Review by joel-hamill — CHANGES_REQUESTED** [review]

File: 1.9/usage/managing-services/private-docker-registry.md (hunk 4 above)
Comment (sascala) [inline 1.9/usage/managing-services/private-docker-registry.md]: I don't know that it's a question of accessibility -- I think when you add the app definition to Marathon, the file will be pulled from wherever on your local machine you specify.

**Comment by sascala** [comment]
@joel-hamill Changes made

**Review by joel-hamill — APPROVED** [review]

**Pull request merged**